src/components/Button/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import "./button.scss";
 import clsx from "clsx";
 
-const Button = ({ text, className, onClick }) => (
-  <button className={clsx(className)} onClick={onClick}>
+const Button = ({ text, className, onClick, type }) => (
```

Comment:
if property has the same name as component attribute you can paste it in component as a lot of props

const Button = ({ text, className,  ...buttonProps }) => (
<button className={clsx(className)} {...buttonProps}>
    {text}
  </button>
)